src/components/PopoverMenu.tsx

```diff
@@ -269,14 +274,14 @@ function PopoverMenu({
                                     shouldShowSelectedItemCheck={shouldShowSelectedItemCheck}
                                     shouldCheckActionAllowedOnPress={false}
                                     onFocus={() => setFocusedIndex(menuIndex)}
-                                    style={{backgroundColor: item.isSelected ? theme.activeComponentBG : undefined}}
+                                    style={{backgroundColor: focusedIndex === menuIndex ? theme.activeComponentBG : undefined}}
```

Comment:
Why is this needed? `PopoverMenu` is used by many components and this change could cause regressions. Let's avoid changing `PopoverMenu` unless it's necessary

Reply:
Because the current logic makes the item with `isSelected` as `true` is always has focused background although we use the keyboard to focus on another option.

Reply:
Let's not introduce changes to `PopoverMenu`. Instead fix this at the account switcher level. It's probably due to the list not being constructed correctly. This change already caused a regression https://github.com/Expensify/App/pull/49051#issuecomment-2352602655.

Reply:
@s77rt 

> This change already caused a regression

The problem is we reset the focusedIndex to `-1` when the modal is hidden but we don't reset it when it's opened again.

https://github.com/Expensify/App/blob/539327dc2d0585624e5e0b990ca0d79f4e66e81b/src/components/PopoverMenu.tsx#L221-L223

This bug also happens here. Then I think we should do the change in `PopoverMenu` level.

https://github.com/user-attachments/assets/76c38ad2-ad49-4c7a-b8dd-7c70845b3dd5

> Instead fix this at the account switcher level

Additionally, I implemented this solution and this requires the change the comparing in the `memo` function (because we need to change the `isSelected` value) which causes the perf-test fail

```
prevProps.menuItems=== nextProps.menuItems
```

Reply:
@nkdengineer Which video? The above one? I'm not sure I see the problem you are referring to, either way let's focus on fixing the account switcher modal

Reply:
> Why we are making this change? The current user should always be marked as selected

@s77rt  In `PopoverMenu`, we display an item with a background based on `item.isSelected`. For example like this video, this component also uses `PopoverMenu` and you can see when we focus on another item, the item with `isSelected` is `true` and still has the focused background that is why I need this change. If it's a bug, please take a look at my comment here otherwise we don't need to change other than the style in `PopoverMenu`

https://github.com/user-attachments/assets/76c38ad2-ad49-4c7a-b8dd-7c70845b3dd5

Reply:
@nkdengineer Let's not handle that here and try to keep the changes minimal for now. (Please do not modify `PopoverMenu` logic either unless we have a well defined problem with the account switcher case)